src/BaGet.UI/package-lock.json

```diff
@@ -7162,7 +7162,8 @@
             },
             "ansi-regex": {
               "version": "2.1.1",
-              "bundled": true
+              "bundled": true,
+              "optional": true
```

Comment:
Let's revert these changes as we're not doing any UI changes as part of this PR.

Reply:
I have no idea how this was changed, honestly. Maybe I was working on the wrong branch or something 😅